File: rust/src/models/o_auth_apps_response.rs
// this file is @generated
use serde::{Deserialize, Serialize};

use super::o_auth_app::OAuthApp;

#[derive(Clone, Debug, Default, PartialEq, Deserialize, Serialize)]
pub struct OAuthAppsResponse {
    pub data: Vec<OAuthApp>,
}

impl OAuthAppsResponse {
    pub fn new(data: Vec<OAuthApp>) -> Self {
        Self { data }
    }
}

File: typescript/src/models/oAuthAppsResponse.ts
// this file is @generated
import { type OAuthApp, OAuthAppSerializer } from "./oAuthApp";

export interface OAuthAppsResponse {
  data: OAuthApp[];
}

export const OAuthAppsResponseSerializer = {
  _fromJsonObject(object: any): OAuthAppsResponse {
    return {
      data: object["data"].map((item: any) => OAuthAppSerializer._fromJsonObject(item)),
    };
  },

  _toJsonObject(self: OAuthAppsResponse): any {
    return {
      data: self.data.map((item: any) => OAuthAppSerializer._toJsonObject(item)),
    };
  },
};
